package.json

```diff
@@ -173,7 +173,8 @@
     "lint": "npm run lint-fmt && npm run lint-eslint",
     "lint-eslint": "eslint .",
     "lint-fmt": "prettier --check .",
-    "test": "node --test out/test/**/*.js",
+    "test": "node --test out/test/**/*.test.js",
+    "scarb-pick-test": "extest setup-and-run './out/test/lsVersion.js' --code_version max --code_settings ui-test/settings.json --extensions_dir .test-extensions",
```

Comment:
why have you put ui-tests in `test` directory?

I think it'd be much more clear to have a directory structure like this, wdyt?:
```
/test
/ui-test/settings.json
/ui-test/generic/*.test.js
/ui-test/scarb/*.test.js
```